### ISSUE TITLE
Unhandled error during execution of watcher callback    at <UserButton> 

### ISSUE THREAD

**Issue by 2322312693** [wobsoriano/vue-clerk#112]

**Comment by 2322312693** [comment]
```vue
<UserButton>
<UserButton.MenuItems>
<UserButton.Action label="Open chat" @click="openChat">
<template #labelIcon>
<Icon />
</template>
</UserButton.Action>
</UserButton.MenuItems>
</UserButton>
```

When I changed it to this it worked fine

**Comment by wobsoriano** [comment]
Which version are you on? It should be fixed in 0.5.1

**Comment by wobsoriano** [comment]
Feel free to reopen if you still encounter the issue
